src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -0,0 +1,27 @@
+# DO NOT CHANGE THESE VALUES
```

Comment:
Please explain why, what they are used for and why, if that happens it is a catastrophe.
Also You can add a simple unit test that checks that the number of strings is the expected one.
So if somebody adds a string, the test will fail and they will adjust the test, annoying but not too bad.
If somebody messes up this and removes a string, the test fails when it should not.

Basically it gives you a safeguard when this changes to ensure your change is deliberate when you make one.